functions/src/export-csv.ts

```diff
@@ -96,43 +93,37 @@ export async function exportCsvHandler(
 
   lois.forEach(loi => {
     const loiId = loi.id;
-    const location = loi.get('location') || {};
     const submissions = submissionsByLocationOfInterest[loiId] || [{}];
     submissions.forEach(submission => {
       const row = [];
-      row.push(loi.get('id') || '');
-      row.push(location['_latitude'] || '');
-      row.push(location['_longitude'] || '');
-      row.push(toWkt(loi.get('geoJson')) || '');
+      // Header: fid
+      row.push(loi.get('properties').id || '');
+      // Header: geometry
+      row.push(toWkt(loi.get('geometry')) || '');
       // TODO(#1288): Clean up remaining references to old responses field
-      const data = submission['data'] || submission['responses'] || submission['results'] || {};
+      const data =
+        submission['data'] ||
+        submission['responses'] ||
+        submission['results'] ||
+        {};
+      // Header: loop over survey
       tasks.forEach((task, taskId) => row.push(getValue(taskId, task, data)));
       csvStream.write(row);
     });
   });
   csvStream.end();
 }
 
-function toWkt(geoJsonString: string) {
-  const geoJsonObject = parseGeoJson(geoJsonString);
-  const geometry = getGeometry(geoJsonObject);
-  return geometry ? geojsonToWKT(geometry) : '';
-}
-
-function parseGeoJson(jsonString: string) {
-  try {
-    // Note: Returns null when jsonString is null.
-    return JSON.parse(jsonString);
-  } catch (e) {
-    return null;
+function toWkt(geometryObject: any): string {
+  // Make sure that the coordinates field is no longer using the Firestore format
+  if (!geometryObject.coordinates) {
+    return '';
   }
-}
+  geometryObject.coordinates = Datastore.fromFirestoreMap(
```

Comment:
I would assume the converter would want the whole `geometryObject` so that it could use the different named fields like `type` to parse the Geometry, roughly as in. https://geojson.org/geojson-spec.html

Extracting the coordinates from the `geometryObject` seems like a job for the converter, not the geometry->wkt conversion function.